src/App.tsx

```diff
@@ -1,10 +1,49 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './component/Header/Header';
+import { Footer } from './component/Footer/Footer';
+import { TodoApp } from './component/TodoApp/TodoApp';
+import { getTodos } from './api/todos';
+import { FilteredBy } from './types/FilteredBy';
 
-const USER_ID = 0;
+const USER_ID = 10610;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredBy, setFilteredBy] = useState<FilteredBy>(FilteredBy.ALL);
+  const [isError, setIsError] = useState(false);
+
```

Comment:
Consider using the state for filtered todos here and update this state after changing the filterBy variable. It could help to render only filtered todos inside your TodoApp component